docs/modules/ROOT/pages/migrating-from-helm.adoc

```diff
@@ -0,0 +1,20 @@
+= Migrating From Helm Chart to the Operator
+
+Unfortunately there isn't any way to migrate without re-creating the cluster as the Operator needs to take control of the custom resources which were created by the Helm chart, which it is not possible.
+
+There are two ways to migrate from a cluster that is created by the Hazelcast Helm Chart.
+
+WARNING: For both solutions to work, `persistence` must be enabled. You should also set `cluster-data-recovery-policy` parameter's value to `PARTIAL_RECOVERY_MOST_COMPLETE` to avoid any issues during restore process.
```

Comment:
```suggestion
WARNING: For either solution to work, `persistence` must be enabled. You should also set the `cluster-data-recovery-policy` parameter's value to `PARTIAL_RECOVERY_MOST_COMPLETE` to avoid any issues during the restore process.
```